src/Microsoft.Management.Deployment/PackageManager.idl

```diff
@@ -280,6 +280,13 @@ namespace Microsoft.Management.Deployment
         /// a lot of fields and no one requesting it.

```

Comment:
nit: Comments related to GetManifest() could probably be removed now as we have started returning PackageInstallerInfo and PackageLocales